samples/data_versioning_mlflow_many_dependent_models/loan_classification.ipynb

```diff
@@ -0,0 +1 @@
pplication/vnd.databricks.v1+cell":{"inputWidgets":{},"nuid":"8dfacc10-8fc7-4572-80a4-9f56d37aa21c","showTitle":false,"title":""}},"outputs":[],"source":["run_id = \"88e43402bf904f4a9de3266ef47236eb\"\n","model_names=['lime_model', 'lr_model']\n","\n","artifact_path = ( f\"abfss://{DEFAULT_ARTIFACT_CONTAINER}@dlsloandev.dfs.core.windows.net/{experiment_id}/{run_id}\" if \n","                 IS_CRED_PASS else \n","                 f'/mnt/{DEFAULT_ARTIFACT_CONTAINER}/{experiment_id}/{run_id}')\n","for model in model_names:\n","  try: \n","    dbutils.fs.ls(f\"{artifact_path}/{model}\")\n","    displayHTML(\"Already copied!\")\n","  except:\n","    displayHTML(f\"Copying {model} to \"f\"{artifact_path}/{model}\")\n","    copy_model(experiment_id,run_id,model_name=model, filepath=f\"{artifact_path}/{model}\")"]},{"cell_type":"code","execution_count":null,"metadata":{"application/vnd.databricks.v1+cell":{"inputWidgets":{},"nuid":"cea111d8-0d08-4db1-9ae8-2dabd6349e84","showTitle":false,"title":""}},"outputs":[],"source":["display(dbutils.fs.ls(artifact_path))"]},{"cell_type":"markdown","metadata":{"application/vnd.databricks.v1+cell":{"inputWidgets":{},"nuid":"9b1ad38e-be9a-49b9-84b0-77d4e79ef906","showTitle":false,"title":""}},"source":["## Re-load a model and test\n","\n","- We use LIME model as an example, re-load it from the artifact path and test it"]},{"cell_type":"code","execution_count":null,"metadata":{"application/vnd.databricks.v1+cell":{"inputWidgets":{},"nuid":"279a5933-9ff2-4334-99b2-5ed32ee7f2ca","showTitle":false,"title":""}},"outputs":[],"source":["test_model = 'lime_model'\n","model_path = f'{artifact_path}/{test_model}'\n","lime_reloaded = PipelineModel.load(model_path)\n","lime_df2 = lime_reloaded.transform(pred_df.select('features'))\n","dfLimeSel = lime_df.select('weights').withColumnRenamed('weights', 'splitcol')\n","dfSplit = splitVector(dfLimeSel, feature_cols )\n","dfResult = dfSplit.drop(\"splitcol\")\n","dfResult.show()"]},{"cell_type":"code","execution_count":null,"metadata":{"application/vnd.databricks.v1+cell":{"inputWidgets":{},"nuid":"436f35c2-7bf8-455d-98b0-271ac2f8ff1f","showTitle":false,"title":""}},"outputs":[],"source":[]}],"metadata":{"application/vnd.databricks.v1+notebook":{"dashboards":[],"language":"python","notebookMetadata":{"pythonIndentUnit":2},"notebookName":"loan_classification","notebookOrigID":1293095669402272,"widgets":{"deltaVersion":{"currentValue":"5","nuid":"02fdaace-53f1-41bd-9862-7a8424d654c6","widgetInfo":{"defaultValue":"6","label":"Table version, default=6","name":"deltaVersion","options":{"validationRegex":null,"widgetType":"text"},"widgetType":"text"}},"tbl_name":{"currentValue":"datalake/lc_loan","nuid":"8066955a-809f-4046-85c6-3f98d4efa1d5","widgetInfo":{"defaultValue":"delta-ds-test/lc_loan","label":"tbl_name,default=delta-ds-test/lc_loan","name":"tbl_name","options":{"validationRegex":null,"widgetType":"text"},"widgetType":"text"}}}},"language_info":{}},"nbformat":4,"nbformat_minor":0}
```

Comment:
Not the easiest to review, these notebooks 😅

"# new_features should be the same length as vector column length" <- is this possible to check in code (and throw an exception if violated)?

Reply:
`run_id = "88e43402bf904f4a9de3266ef47236eb"`
This seems awfully GUID-y to be checked in.

Reply:
the first one, it will throw exception as by default will do schema verification. too expensive to add extra checking on the length..
run_id removed and added a comment

Reply:
Okay I'm fine with that (re: no checking)